corehq/apps/api/tests.py

```diff
@@ -143,15 +143,8 @@ def tearDownClass(cls):
         cls.api_key.delete()
         cls.user.delete()
 
-        SubscriptionAdjustment.objects.all().delete()
-
-        if cls.subscription:
-            cls.subscription.delete()
-
-        if cls.account:
-            cls.account.delete()
-
         for domain in Domain.get_all():
+            Subscription._get_active_subscription_by_domain.clear(Subscription, domain.name)
```

Comment:
Won't this happen automatically during domain deletion: https://github.com/dimagi/commcare-hq/blob/d4abc929b02fdf9bba719dbca80fc047cd90db03/corehq/apps/domain/deletion.py#L105

Reply:
I would've thought so too, but travis kept getting this failure:
https://api.travis-ci.org/v3/job/441944795/log.txt

Since it seems specific to EWSLocationResourceTest, I figured I would just patch up the test.